### ISSUE TITLE
Send a verification email reminding of Palo Alto IP Change

### ISSUE THREAD

**Issue by MauriceReeves-usds** [CDCgov/prime-reportstream#2157]

**Comment by Adrian-Brewster** [comment]
First Email:

Hello and Happy Friday! 

ReportStream is continuously working to monitor potential risks to data quality and transport security as we deliver better, faster, and more accurate COVID-19 messages to your state.  In light of the recent SolarWinds hack, we at ReportStream have decided to move outbound traffic from Azure through a security firewall, which means it will be coming from a different IP address.  This change is scheduled for September 30th and will require you to expand your whitelisting range to include 158.111.236.0/24 to continue receiving data from us.  

Please let me know if this will be an issue for you or if you have any questions.

**Comment by Adrian-Brewster** [comment]
Follow up email:

Email Header : Palo Alto Firewall Migration - Feedback Required 
Flag as urgent

Hello all,

This email is a reminder that ReportStream will be  moving our IP address from an Azure-based IP range to a CDC-based IP range starting September 30th.  Please whitelist the following IP Range to prevent any disruptions in our message flow: 158.111.236.0/24. Once the migration is complete, you will receive an email letting you know that you can remove our old IP address from your firewall.  

Please respond to this email confirming that the necessary changes have been made. Thanks!

**Comment by mlighdc** [comment]
@Adrian-Brewster some tweaks. Thanks.

- Palo Alto Firewall Migration seems very internal to us
- We would like them to take an action so I change the email to Action Required
- I would also add the reportstream@cdc.gov for support 


Email Header : **Action Required - Please Whitelist New IP Range for PRIME ReportStream**
Flag as urgent

Hello all,

This email is a reminder that PRIME ReportStream will be moving our IP address from an Azure-based IP range to a CDC-based IP range starting September 30th, 2021. If you have not done so already, please whitelist the following IP range to prevent any disruptions to your message flow: **158.111.236.0/24.** 

Once the migration has been completed, we will send another email to let you know that you can remove our old IP address from your firewall.

Please respond to this email confirming that the necessary changes have been made. If you have any questions, please let us know. Thanks!

**Comment by Adrian-Brewster** [comment]
Email Header : **Prime ReportStream IP Range Migration Postponed - New Date October 21st, 2021**

Hello all,

Prime ReportStream is moving our deployment date to October 21st, 2021 to migrate from an Azure-based IP range to a CDC-based IP range.  If you have not done so already, please whitelist the following IP range to prevent any disruptions to your message flow: 158.111.236.0/24 and respond to this email to let us know this is complete.  If you have already completed this, no additional steps are required at this time.

We will send another email once the migration is complete to let you know that you can remove our old IP address from your firewall. 

Thank you and have a wonderful weekend!

**Comment by mlighdc** [comment]
@Adrian-Brewster see suggested edits below. Thanks.

******************************************************
Email Header : Update - PRIME ReportStream IP Range Migration has Been Postponed until October 21, 2021

Hello all,

To allow for additional testing, we have decided to postpone the PRIME ReportStream IP migration from from an Azure-based range to a CDC-based range. The new date for the IP migration is **October 21, 2021.**

If you have already whitelisted the new IP range, no additional action is needed at this time. If you have not done so, please whitelist the following IP range to prevent any disruptions to your message flow: **158.111.236.0/24**. We would appreciate if you can notify us when it is completed by responding to this email. 

We will send another email once the migration is complete to let you know that you can remove our old IP address from your firewall.

Thank you and have a wonderful weekend!

**Comment by Adrian-Brewster** [comment]
Email Header: Update: PRIME ReportStream IP Migration Complete

Hello All,

This email is to inform you that the the PRIME ReportStream IP migration from an Azure-based range to a CDC-based range is now complete and you may remove our old IP address from your firewall.

Thanks! 

@mlighdc I figured we could keep this short and simple.  If you have anything you would like to add, feel free to do so!

**Comment by mlighdc** [comment]
@Adrian-Brewster I would make the following tweak to make sure folks know it is an update and what to do if they experience any issues. There was also a slight typo. Please make sure to copy me on the email. Thanks!

email Header: **Update -** PRIME ReportStream IP Migration Has Been Completed

Hello All,

This email is to inform you that the PRIME ReportStream IP migration from an Azure-based range to a CDC-based range has been completed. If your organization whitelists IP addresses, please remove our old IP address from your firewall. 

If you encounter any issues or have any questions about the IP transition, please contact us at reportstream@cdc.gov.

Thank you for patience during the IP transition.

The PRIME ReportStream Team
